packages/site/pages/components/note.js

```diff
@@ -0,0 +1,240 @@
+        linking straight to the content.
+      </P>
+      <Example.React
+        themeToggle
+        includes={{ Avatar, Note, Icon }}
+        codes={[
+          `
+<Note
+  actionBar={[
+    <Note.Action icon={<Icon id={Icon.ids.bookmark} />} title="Bookmark" />,
+    <Note.Action icon={<Icon id={Icon.ids.more} />} title="More" />
+  ]}
+  avatar={<Avatar name="Mark Twain" src="//placebear.com/128/128" />}
+  heading="Mark Twain"
+  message={(
+    <p>Customs do not concern themselves with right or wrong or reason. But they have to be obeyed; one reasons all around them until he is tired, but he must not transgress them, it is sternly forbidden.</p>
+  )}
+  metadata={['Lorem ipsum dolor sit amet', 'Curabitur dolor sapien']}
+/>
+        `
+        ]}
+      />
+
+      <SectionHeading>Without Attribution</SectionHeading>
+      <P>
+        A note can also be used without author attribution. The actions will be
+        moved to the bottom next to the metadata.
+      </P>
+      <Example.React
+        themeToggle
+        includes={{ Note }}
+        codes={[
+          `
+<Note
+  actionBar={[
+    <Note.Action icon={<Icon id={Icon.ids.bookmark} />} title="Bookmark" />,
+    <Note.Action icon={<Icon id={Icon.ids.more} />} title="More" />
+  ]}
+  message={(
+    <p>Customs do not concern themselves with right or wrong or reason. But they have to be obeyed; one reasons all around them until he is tired, but he must not transgress them, it is sternly forbidden.</p>
+  )}
+  metadata={['Lorem ipsum dolor sit amet', 'Curabitur dolor sapien']}
+/>
+        `
+        ]}
+      />
+      <SectionHeading>List</SectionHeading>
+      <P>Use a List when display a group of related Notes.</P>
+      <Example.React
+        themeToggle
+        includes={{ Note }}
+        codes={[
+          `
+<Note.List>
+  <Note
+    avatar={<Avatar name="Mark Twain" src="//placebear.com/128/128" />}
+    heading="Mark Twain"
+    message={(
+      <p>Customs do not concern themselves with right or wrong or reason. But they have to be obeyed; one reasons all around them until he is tired, but he must not transgress them, it is sternly forbidden.</p>
+    )}
+  />
+
+  <Note
+    avatar={<Avatar name="Samuel Lanhorne Clemens" src="//placebear.com/128/128" />}
+    heading="Samuel Lanhorne Clemens"
+    message={(
+      <p>There are those who imagine that the unlucky accidents of life—life's 'experiences'—are in some way useful to us. I wish I could find out how. I never know one of them to happen twice. They always change off and swap around and catch you on your inexperienced side.</p>
+    )}
+  />
+
+  <Note
+    avatar={<Avatar name="Mark Twain" src="//placebear.com/128/128" />}
+    heading="Mark Twain"
+    message={(
+      <p>Clothes make the man. Naked people have little or no influence on society.</p>
+    )}
+  />
+</Note.List>
+`
+        ]}
+      />
+    </Content>
+  </Chrome>
+))
```

Comment:
Even more mysterious, when I refresh the `/components/note` page, I get nextjs not rendering, but giving this:

```
(0 , _psDesignSystemPropTypes.arrayOfMaxLength) is not a function
TypeError: (0 , _psDesignSystemPropTypes.arrayOfMaxLength) is not a function
    at Object.<anonymous> (/Users/jaketrent/dev/design-system/packages/note/dist/react/index.js:261:60)
    at Module._compile (internal/modules/cjs/loader.js:723:30)
    at Module._compile (/Users/jaketrent/dev/design-system/node_modules/source-map-support/source-map-support.js:492:25)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:734:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Module.require (internal/modules/cjs/loader.js:659:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/jaketrent/dev/design-system/packages/note/react.js:1:18)
```